rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -334,6 +336,22 @@
           }
         }
       }
+    },
+    "blocked": {
+      "title": "Blocking",
```

Comment:
Could the title be `Blocked` instead of `Blocking`?

Reply:
The PO asked this title to be "Blocking".

Reply:
@blankoworld @iGormilhit Yes I asked to modify it. But for long term maintenance, it is maybe smarter to adapt also the property name? This is up to the dev team to decide.